HTML/SimpSquad.html

```diff
@@ -0,0 +1,42 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <title>SimpSquad — Keep Talking and Nobody Explodes Module</title>
+    <meta name="viewport" content="initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+</head>
+<body>
+    <div class="section">
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">SimpSquad</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/SimpSquad.svg" class="diagram">
+                <h2>On the Subject of SimpSquad</h2>
+                <p class="flavour-text">Flavor text fdhusjifkahusdjfkahujdskfiasik</p>
+                </br>
+                <p>This module will present you with a small red button. </br></br>
+                   If there are multiple SimpSquads on the bomb. Only one of these buttons will be red. The rest will be blue.</br>
+                <h4>Solving the blue SimpSquads:</h4></br>
+                   Each blue SimpSquad is linked to another blue SimpSquad. Pressing a blue SimpSquad will solve the module that it is linked to. </br>
+                <h4>Solving the red SimpSquad:</h4></br>
+                   With multiple SimpSquads on a bomb, there will be several “chains”, series of blue SimpSquads that are linked to each other. </br></br>
+                   To follow a chain, first press a blue SimpSquad and note down which module it solves. Press the button on that module and observe which module it solves. Continue this until the initially pressed SimpSquad is solved. </br><strong>This will form one chain.</strong></br></br> 
+                   Note that in the event that pressing a blue SimpSquad solves itself, that will still equal a chain of length 1.</br></br>
+                   To solve the red SimpSquad, first solve all other blue SimpSquads and note down the number of chains within the bomb modulo 100. </br></br>
+                   Press the red SimpSquad when the digits of the bomb’s countdown timer contain this number.</br></br></br></br>
+ 
+                   In the event that there is only one SimpSquad on the bomb, there are no chains present, so press the Button when a 0 is present in the countdown timer.</p>
+</br></br><p class="flavour-text">Special thanks to Asew54321, miki2003pl, and Danny7007 for deducing the manual.</p>
```

Comment:
This spacing definitely needs to be cleaned up. In this file as a whole, make sure all opening tags have a closing tag that there's no unnecessary tags.